drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -299,13 +300,22 @@ function DragAndDropTemplates(configuration) {
             attemptsUsedInfo = h("span.attempts-used", {id: attemptsUsedId}, attemptsUsedText);
         }
 
+        var submitSpinner = null;
+        if (ctx.submit_spinner) {
+            submitSpinner = h('span', [
+                h('span.fa.fa-spin.fa-spinner', {attributes: {'aria-hidden': true}}),
+                h('span.sr', gettext('Saving'))
+            ]);
+        }
+
         return (
             h("div.action-toolbar-item.submit-answer", {}, [
                 h(
                     "button.btn-brand.submit-answer-button",
                     submitButtonProperties,
                     [
-                        (ctx.submit_spinner ? h("span.fa.fa-spin.fa-spinner") : null),
+                        submitSpinner,
+                        ' ',  // whitespace between spinner icon and text
                         gettext("Submit")
```

Comment:
In standard mode the button is not used at all, but it is *always* visible in assessment mode. The spinner icon is only visible while the submit XHR is in progress after the user clicks the button, though. While the XHR is in progress and the spinner visible, the button is also disabled (see screenshots). The spinner gets hidden as soon as the XHR is complete.

![screen shot 2016-12-23 at 08 37 19](https://cloud.githubusercontent.com/assets/32585/21449231/5bad9592-c8eb-11e6-85fb-76f71676abb8.png)
![screen shot 2016-12-23 at 08 37 53](https://cloud.githubusercontent.com/assets/32585/21449232/5bae919a-c8eb-11e6-9d12-b9cd382ae4c2.png)

The "Submit" text is always visible, but the spinner icon indicates that submission is in progress, so the icon technically still needs sr text. The sr text currently says "Submitting", which is fine on its own, but screen readers are going to read "Submitting submit", which is not ideal. The XHR typically completes pretty quickly, so it is not very likely users will notice that text, but I would appreciate better ideas as to what sr text to put next to the spinner.